sdk/storage/azure-storage-blob/azure/storage/blob/_models.py

```diff
@@ -651,7 +651,7 @@ def __init__(self, **kwargs):
         self.source = kwargs.get('x-ms-copy-source')
         self.status = get_enum_value(kwargs.get('x-ms-copy-status'))
         self.progress = kwargs.get('x-ms-copy-progress')
-        self.completion_time = kwargs.get('x-ms-copy-completion_time')
+        self.completion_time = kwargs.get('x-ms-copy-completion-time')
```

Comment:
This is a great fix. Thanks! If you could add this to the `blobs` package CHANGELOG under `Bugs Fixed`
"- Fixed a typo in the initialization of `completion_time` for the `CopyProperties` model."

With that addition, we will likely be able to take this pull-request as is. Thanks for your community contribution!

Reply:
@vincenttran-msft Just added the entry. Let me know if that works :)

Reply:
Thanks, left a response. Also I may have caused a merge conflict in the changelog (merged a different PR with a changelog entry) so you may need to resolve that!